packages/data-explorer-ui/src/components/Layout/components/Header/components/ProfileComponent/components/authenticationMenu.tsx

```diff
@@ -0,0 +1,65 @@
+import { IconButton, Menu, MenuItem } from "@mui/material";
+import React, { MouseEvent, useState } from "react";
+import { UserProfile } from "../../../../../../../providers/authentication";
+import * as Styled from "./authenticationMenu.styles";
+
+export interface AuthenticationMenuProps {
+  menuPosition: "left" | "right";
+  onLogout: () => void;
+  userProfile: UserProfile;
+}
+
+export const AuthenticationMenu = ({
+  menuPosition = "left",
+  onLogout,
+  userProfile,
+}: AuthenticationMenuProps): JSX.Element => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLButtonElement>(null);
+  const open = Boolean(anchorEl);
+  const growTo = menuPosition === "left" ? "right" : "left";
+
+  const onOpenMenu = (event: MouseEvent<HTMLButtonElement>): void => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const onCloseMenu = (): void => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <>
+      <IconButton onClick={onOpenMenu}>
+        <Styled.Avatar
```

Comment:
Let's import all styled components directly and render as `Avatar` rather than `Styled.Avatar` (keeping the package "language" consistent) :)